Add a comment where to find the template file which should
be used as the basis where to apply the patch series at.

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -3,6 +3,8 @@
 # $Id: make.conf 13289 2009-04-03 22:27:18Z zmedico $
 # Contains local system settings for Portage system
 
+# Keep in sync with: /usr/share/portage/config/make.conf.example
+#
 # Please review 'man make.conf' for more information.
 
 # Build-time functionality
